app/assets/javascripts/calendars/holidays.es6

```diff
@@ -24,27 +24,10 @@
       super(el, calendarConfig);
     }
 
-    eventRender(event, element) {
-      const $button = $(`
-        <button class="close t-delete-holiday">
-          <span aria-hidden="true">X</span><span class="sr-only">Remove slot</span>
-        </button>
-      `);
-
-      $button.on('click', () => {
-        if (confirm("Are you sure you want to delete this event?")) {
-          $.ajax({
-              type: "DELETE",
-              url: this.$el.data('holidays-path') + '/?holiday_ids=' + event.holiday_ids,
-              dataType: "json",
-              complete: () => {
-                this.$el.fullCalendar('removeEvents', event._id);
-              }
-          });
-        }
-      });
-
-      element.append($button);
+    eventClick(event) {
+      if (!event.bank_holiday) {
```

Comment:
Can we not create a `url` attribute in the holidays feed? Then this method can be removed